src/store/wizardSlice.ts

```diff
@@ -37,6 +42,10 @@ type wizardState = {
     registrationType: string;
     activationKey: ActivationKeys['name'];
   };
+  details: {
+    blueprintName: ComposeRequest['image_name'] | '';
+    blueprintDescription: ComposeRequest['image_description'] | '';
```

Comment:
Same comment as above, we can remove the  `| ''`, the type of `ComposeRequest['image_name']` is already `string | undefined` so `''` is already taken care of.